src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/utils.ts

```diff
@@ -112,6 +137,32 @@ export const getThorchainSaversPosition = async ({
   const poolAssetId = assetIdToPoolAssetId({ assetId })
 
   const accountPosition = await (async () => {
+    if (assetId === thorchainAssetId) {
+      const { data: runepoolInformations } = await queryClient.fetchQuery({
+        queryKey: ['thorchainRunepoolOpportunity', accountId],
+        queryFn: () =>
+          axios.get<ThorchainFromAddressRunepoolInformationsResponseSuccess>(
+            `${getConfig().REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/rune_provider/${address}`,
+          ),
+      })
+
+      const runepoolOpportunity: ThorchainSaverPositionResponse = {
+        asset: 'THOR.RUNE',
+        asset_address: fromAssetId(thorchainAssetId).assetReference,
```

Comment:
```suggestion
        asset_address: address,
```

Reply:
Or alternatively `runepoolInformations.rune_address`

Reply:
`address` is wrong, ok with `runepoolInformation.rune_address`

Changed in [cb8c726](https://github.com/shapeshift/web/pull/7436/commits/cb8c726c4618f4cef890ee41b1dbd3c95e1fe658)